q2_health_index/_gmhi.py

```diff
@@ -47,10 +48,55 @@ def calculate_gmhi(ctx,
 
     # TODO Valentyn: CALCULATE GMHI (BELOW WE ARE JUST LOADING EXPECTED DATA !!!!)
     expected = os.path.join(os.path.dirname(__file__), 'tests/data/expected/4347_final_gmhi.tsv')
-    gmhi = pd.read_csv(expected, sep='\t', index_col=0, header=0, squeeze=True)
+    gmhi_in = pd.read_csv(expected, sep='\t', index_col=0, header=0, squeeze=True)
+
+    # Removing unclassified and virus species - skipped transposing steps
+    na_species = gmhi_in.index.str.contains('unclassified|virus', regex=True)
+    species_profile_2 = gmhi_in[~na_species]
+
+    # Re-normalization of species' relative abundances after removing unclassified and virus species
+    species_profile_3 = species_profile_2.apply(lambda x: x / x.sum(), axis=0)
+    species_profile_3[species_profile_3 < 0.00001] = 0
```

Comment:
are you sure you want to hard-code those values? maybe it would be more transparent to save it as a variable?
also I would recommend to express such small floats using the scientific notation, ie. `0.00001 == 1e-5` - easier to spot check

Reply:
That's a good point, I think adding it as a parameter to the function is the best solution, as someone might want to dig deeper.